donor-registry/src/app/forms/forms.component.ts

```diff
@@ -259,7 +259,7 @@ export class FormsComponent implements OnInit {
           "mobileNumber": this.model["emergencyDetails"]['mobileNumber'],
         },
         "registrationBy": (this.model["registrationBy"]) ? this.model["registrationBy"]: "mobile",
-        "mobileno": (this.model["mobileno"])? this.model["mobileno"] : {},
+        // "mobileno": (this.model["mobileno"])? this.model["mobileno"] : {},
```

Comment:
@suvarnakale Insted of commented,  remove this line

Reply:
okay